types/meteor/mongo.d.ts

```diff
@@ -219,6 +240,8 @@ declare module "meteor/mongo" {
             map<M>(callback: (doc: U, index: number, cursor: Cursor<T, U>) => M, thisArg?: any): Array<M>;
             observe(callbacks: ObserveCallbacks<U>): Meteor.LiveQueryHandle;
             observeChanges(callbacks: ObserveChangesCallbacks<T>, options?: { nonMutatingCallbacks?: boolean }): Meteor.LiveQueryHandle;
+            _mongo: MongoInternals.Connection;
+            _synchronousCursor: MongoInternals.SynchronousCursor;
```

Comment:
Can't we use the `mongodb` package here?

Reply:
Can you please elaborate? My sole purpose of adding those types is such code block would yield no errors:

```
const connection = await MongoClient.connect(
      MongoInternals.defaultRemoteCollectionDriver().mongo.client.s.url,
      {
        useNewUrlParser: true,
        useUnifiedTopology: true,
        poolSize: 1,
      }
    );

    const db = await connection.db(
      MongoInternals.defaultRemoteCollectionDriver().mongo.client.s.options
        .dbName
    );
```

Reply:
Of course. What I meant was to not type missing properties ad hoc, but rather use typings of the `mongodb` package (imported at the top of the file) instead.

Reply:
Thanks for your response <3. Please bear with me.
`mongodb` package lacks explicit `Connection` type. The closest thing we have to it is `Collection` interface given both have `find` and `findOne` properties but even then, it still lacks important keys like `mongo`, `db`, `_oplogHandle` and `_createSynchronousCursor`. I presume these properties are added by [meteor](https://github.com/meteor/meteor/blob/46e00a875726dff078c59eaf7fdc7b956cf0a996/packages/mongo/mongo_driver.js#L201) which makes it unpractical to have them exported from `mongodb`

But as for `synchronousCursor`, `Mongo.Cursor` may be a sufficient alternative. Please double check with me.

Reply:
Indeed, a few of these methods, like [`_createSynchronousCursor` are indeed Meteor-specific](https://github.com/meteor/meteor/blob/46e00a875726dff078c59eaf7fdc7b956cf0a996/packages/mongo/mongo_driver.js#L982). But I don't like the idea of typing _such_ internal code. Wouldn't `mongo` and `db` (created [here](https://github.com/meteor/meteor/blob/46e00a875726dff078c59eaf7fdc7b956cf0a996/packages/mongo/mongo_driver.js#L249-L251)) be enough? You can always `await` asynchronous methods as well as use `for await of` for cursors. Does it fit your needs?

Reply:
Thanks for the great tip! The previous code block just got shortened to:
```
    const { db } = MongoInternals.defaultRemoteCollectionDriver().mongo;
```
 with no need to use `mongodb` npm driver package.
 
 But still I've 
 ```
 Module '"meteor/mongo"' has no exported member 'MongoInternals'.
 ```
 Since I'm importing `MongoInternals`
 
 ```
 import { MongoInternals } from 'meteor/mongo';
```

So, I'll see what we cut from the PR.